go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/frankban/quicktest v1.11.3 // indirect
 	github.com/fsouza/go-dockerclient v1.7.0
 	github.com/go-kit/kit v0.9.0
-	github.com/golang/protobuf v1.3.3
+	github.com/golang/protobuf v1.5.2
```

Comment:
We have been stuck at protobuf v1.3 and older grpc due to etcd issues. See background in https://jira.hyperledger.org/browse/FAB-18363 and https://github.com/hyperledger/fabric/pull/2185.

@guoger has been looking at etcd update in https://github.com/hyperledger/fabric/pull/2707 and may be able to fill in the details.